cmd/fetch.go

```diff
@@ -18,7 +18,18 @@ func Fetch(ctx *cli.Context) error {
 	}
 	client := api.NewClient(c)
 
-	problems, err := client.Fetch(ctx.Args())
+	args := ctx.Args()
+	problems, err := client.Fetch(args)
```

Comment:
We shouldn't perform this fetch if the `all` flag is set to true. What if we moved it to the `else` of the following `if`?